dask_expr/io/parquet.py

```diff
@@ -192,7 +192,11 @@ def to_parquet(
     from dask_expr._collection import new_collection
     from dask_expr.io.parquet import NONE_LABEL, ToParquet
 
-    engine = _set_parquet_engine(meta=df._meta)
+    engine = kwargs.pop("engine")
+    if engine == "fastparquet":
+        raise NotImplementedError("Fastparquet engine is not supported")
+
+    engine = _set_parquet_engine(engine=engine, meta=df._meta)
```

Comment:
This seems fine. We could also include `engine=None` in the explicit argument list above and/or to move the "fastparquet" check into `_set_parquet_engine`.

Reply:
I like this